com.unity.multiplayer.mlapi/Runtime/Transports/Tasks/SocketTask.cs

```diff
@@ -188,12 +193,6 @@ public class SocketTask
         /// Converts to a SocketTasks.
         /// </summary>
         /// <returns>The tasks.</returns>
-        public SocketTasks AsTasks()
-        {
-            return new SocketTasks()
-            {
-                Tasks = new SocketTask[] { this }
-            };
-        }
+        public SocketTasks AsTasks() => new SocketTasks { Tasks = new[] { this } };
```

Comment:
I'm not exactly sure this is clearer, but this syntax of where you can call new[] and the type of `this` is implied is unfamiliar to c++ programmer like me

Reply:
makes sense, I will convert `new[] { this }` to `new SocketTask[] { this }` inline